File: Java/SolarSystem/app/src/main/java/become/sk/solarsystem/Planet.java
package become.sk.solarsystem;

/**
 * Created by peterdruska on 02/08/2017.
 */

public class Planet {
    public final String id;
    public final String name;
    public final Double distanceFromSun; // in AU
    public final Double inhabitants; // in bilions

    public Planet(String id, String name, Double distanceFromSun, Double inhabitants) {
        this.id = id;
        this.name = name;
        this.distanceFromSun = distanceFromSun;
        this.inhabitants = inhabitants;
    }

    @Override
    public String toString() {
        String planetDescription = name + " has " + inhabitants +" billions of inhabitants.";
        return planetDescription;
    }
}

File: Kotlin/SolarSystem/app/src/main/java/become/sk/solarsystem/Planet.kt
package become.sk.solarsystem

/**
 * Created by peterdruska on 02/08/2017.
 */

class Planet(val id: String,
             val name: String,
             val distanceFromSun: Double?, // in AU
             val inhabitants: Double? // in bilions
) {

    override fun toString(): String {
        val planetDescription = "$name has $inhabitants billions of inhabitants."
        return planetDescription
    }
}